src/app/components/NavigationButton/NavigationButton.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+import styled from 'styled-components'
+
+type NavigationButtonProps = {
+  children: ReactNode
+}
+export default function NavigationButton({
+  children,
+}: NavigationButtonProps): JSX.Element {
+  return <NavigationButtonContainer>{children}</NavigationButtonContainer>
+}
+const NavigationButtonContainer = styled.div`
+  height: 70px;
+  width: 70px;
+  background-color: #d9d9f3;
+  border-radius: 3em;
```

Comment:
Do you use em or px in your project (view at line 14 and 15 as well)?
I think it is better to use one of these, so maybe just use em for height and width?

Reply:
I agree `px` shouldn't be used here